src/linker/Linker/AssemblyUtilities.cs

```diff
@@ -39,5 +40,16 @@ public static TypeDefinition FindType (this AssemblyDefinition assembly, string
 			var type = assembly.MainModule.GetType (fullName);
 			return type?.Resolve ();
 		}
+
+		// Takes a documentation signature (not including the documentation member type prefix) and resolves it to a type
+		// in the assembly.
+		public static TypeDefinition FindTypeByDocumentationSignature (this AssemblyDefinition assembly, string signature)
```

Comment:
I don't think this should be here, it won't be universally used and you have to prefix it with DocumentationSignature name already

Reply:
`DynamicDependency` doesn't require the prefix - but I will move it to the signature parser and we can always move it back if it is commonly used.